src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -14,6 +15,8 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
+import static com.minecolonies.api.util.constant.Suppression.*;
+
 //TODO extend list/collection
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)